script/selector-diff-report

```diff
@@ -1,26 +1,30 @@
 #!/bin/bash
 set -e
 
-module=${1:-@primer/css}
-version=${2:-latest}
-
 function log() {
     echo "$@" 1>&2
 }
 
-log "Pulling the latest ${module}/build/data.json ..."
-curl -sL "https://unpkg.com/${module}@${version}/build/data.json" > before.json
+# TODO: update this to pull from @primer/css
+old_path="primer/build/data.json"
+log "Pulling the old $old_path ..."
+curl -sL "https://unpkg.com/$old_path" > before.json
```

Comment:
Not to be down on unpkg, but some future version I'd love to see stuff pulled from github tags (or releases)